package.json

```diff
@@ -17,8 +17,8 @@
   },
   "dependencies": {
     "@alephium/token-list": "^0.0.12",
-    "@alephium/walletconnect-provider": "0.27.5",
-    "@alephium/web3": "0.27.5"
+    "@alephium/walletconnect-provider": "^0.27.5",
```

Comment:
As we agreed, we should fix the versions of all WC related packages to avoid issues we had